src/main/js/distribution/Index.js

```diff
@@ -274,8 +274,8 @@ class Index extends Component {
         if (jobs) {
             jobs.forEach((job) => {
                 if (job && job.fieldModels) {
-                    const channelModel = job.fieldModels.find(fieldModel => fieldModel.descriptorName.startsWith('channel_'));
-                    const providerModel = job.fieldModels.find(fieldModel => fieldModel.descriptorName.startsWith('provider_'));
+                    const channelModel = job.fieldModels.find(fieldModel => FieldModelUtilities.hasKey(fieldModel, KEY_NAME));
```

Comment:
KEY_NAME maps to the common name for the distribution.  That isn't channel or provider specific.  I think you want to import and use KEY_CHANNEL_NAME, and KEY_PROVIDER_NAME.